### ISSUE TITLE
about  "mmdetection/configs/_base_/datasets/_base_/datasets/voc0712.py"

### ISSUE THREAD

**Issue by baibaidedongdong** [open-mmlab/mmdetection#3348]
I have just use the project for a few days,when I try to train faster-rcnn by using VOC datasets,I found in the  line 47 of "mmdetection/configs/_base_/datasets/_base_/datasets/voc0712.py" ,it is  "ann_file=data_root + 'VOC2007/ImageSets/Main/text.txt",dosen't it should be"ann_file=data_root + 'VOC2007/ImageSets/Main/train.txt".

![image](https://user-images.githubusercontent.com/54987492/87867217-7eb4c700-c9bd-11ea-9ea2-e19d43f5be0c.png)

**Comment by ZwwWayne** [comment]
Conventionally we use the test set for validation rather than training set.

**Comment by baibaidedongdong** [comment]
ok,thanks




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Wenwei Zhang"<notifications@github.com&gt;; 
发送时间: 2020年7月19日(星期天) 下午5:19
收件人: "open-mmlab/mmdetection"<mmdetection@noreply.github.com&gt;; 
抄送: "LURENJIA"<1716838988@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [open-mmlab/mmdetection] about  &quot;mmdetection/configs/_base_/datasets/_base_/datasets/voc0712.py&quot; (#3348)





 
Conventionally we use the test set for validation rather than training set.

**Comment by Icecream-blue-sky** [comment]
> Conventionally we use the test set for validation rather than training set.
Hi, @ZwwWayne 
I'm really curious about why ‘data_root + 'VOC2007/ImageSets/Main/trainval.txt' ’ in train ann_file key is repeated twice??
